seed/challenges/03-back-end-development-certification/git.json

```diff
@@ -8,67 +8,27 @@
-        "Install <code>git-it</code> with this command: <code>npm install -g git-it</code>",
-        "Before you start, read <a href='https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/' target='_blank'>Adding a new SSH key to your GitHub account</a> if you have two-factor authentication set up on your GitHub account.",
-        "Now start the tutorial by running <code>git-it</code>.",
-        "Note that you can resize the c9.io's windows by dragging their borders.",
-        "Make sure that you are always in your project's \"workspace\" directory. You can always navigate back to this directory by running this command: <code>cd ~/workspace</code>.",
-        "You can view this Node School module's source code on GitHub at <a href='https://github.com/jlord/git-it' target='_blank'>https://github.com/jlord/git-it</a>.",
-        "Complete \"Get Git\"",
-        "Complete \"Repository\"",
-        "Complete \"Commit to it\"",
-        "Complete \"Githubbin\"",
-        "Complete \"Remote Control\"",
-        "Complete \"Forks and Clones\"",
-        "Complete \"Branches aren't just for Birds\"",
-        "Complete \"It's a Small World\"",
-        "Complete \"Pull, Never out of Date\"",
-        "Complete \"Requesting you Pull, Please\"",
-        "Complete \"Merge Tada!\"",
+        "To complete this challenge, you will need to install a few things on your computer - a text editor, Git, and the Git-It-Electron desktop app.",
+        "Text Editor - A text editor is a good tool to have as a programmer. You can download and install <a href=\"https://atom.io/\" target=\"_blank\">Atom</a>, <a href=\"https://www.sublimetext.com/3\" target=\"_blank\">Sublime Text</a>, or <a href=\"http://brackets.io/\" target=\"_blank\">Brackets</a>. All these editors are available for Windows, Mac and Linux platforms. Alternatively, if you already have a text editor, or you would like to use a different one, you're welcome to do so.",
+        "Git - Download and install Git from <a href=\"https://git-scm.com/download/\" target=\"_blank\">git-scm</a>. You can complete the installation with the recommended defaults. Note - for Windows users - as the installation completes, on the final screen, click on \"Launch Git Bash\". Mac and Linux users may use their terminal instead.",
+        "Git-It-Electron - Visit the <a href=\"https://github.com/jlord/git-it-electron\" target=\"_blank\">git-it-electron</a> repository on Github and <a href=\"https://github.com/jlord/git-it-electron/releases\" target=\"_blank\">download the release</a> corresponding to your operating system. Once the download completes, extract the zipped folder and run the \"Git-it\" application. This will launch the Git-it desktop app, where you will complete the challenges. Once the app launches, click  on \"Start challenge one\".",
```

Comment:
Nit: there is some inconsistent casing here (Git-it vs Git-It)

Reply:
@systimotic I think this should be `Git-it` wherever mentioned. So the first little bit should be "Git-it Electron", I believe.